### DISCUSSION TITLE
I want to use it in source code, not gui.

### DISCUSSION THREAD

**Discussion by harmlessman** [Anjok07/ultimatevocalremovergui#1534] [Q&A]
I want to use it in source code, not gui.

I'd like to use vocal remover for other projects. I think the source code here is about using GUI. Is there usage using CLI or vocal remover? I tried to interpret UVR.py but it's too complicated to understand.

**Comment by Kbass85** [comment]
I've been using this one for CLI.
https://github.com/nomadkaraoke/python-audio-separator
